operator/crd/crd.yaml

```diff
@@ -171,25 +171,45 @@ spec:
                       description: "Configure HTTP server TLS connections using a JKS or PKCS12 keystore. The keystore password should be stored in a secret and referenced in the route Custom Resource. The format of the secret should be `password=<password>`."
                       type: object
                       properties:
-                        secretName:
-                          type: string
-                        key:
-                          type: string
-                        type:
```

Comment:
:question: To stay consistent, should we update the `truststore` field to use the same updated format as `keystore`?

Reply:
Yes, I wrote a follow-on ticket.